### PR TITLE
[MDS-375] Tag v2

### PR DESCRIPTION
**Pull request by tkullisaar**

### CODE REVIEW

File: lib/moon/design/tag.ex [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  slot(default)
+  slot(left_icon)
+  slot(right_icon)
+
+  def render(assigns) do
+    ~F"""
+    <div class={
+      merge([
+        "flex items-center rounded-moon-i-xs select-none text-gohan bg-bulma",
+        set_letter_spacing(@size),
```
Comment (phgrey) [inline lib/moon/design/tag.ex:19]: i'm not sure that having this lot of functions is a good idea? isn't it more transparent to put as clear conditional class list depending on `@size` attriobute?

File: lib/moon/design/tag.ex [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    end
+  end
+
+  defp set_y_padding(size, left_icon, right_icon) do
+    case size do
+      "2xs" ->
+        "py-0"
+
+      _ ->
+        "#{((right_icon || left_icon) && "py-1") || "py-1"}"
```
Comment (phgrey) [inline lib/moon/design/tag.ex]: isn't equal to "py-1" ?

Comment (phgrey) [reply lib/moon/design/tag.ex]: mean don't care if right or left icon is set

File: lib/moon/design/tag.ex (hunk 1 above)
Comment (tkullisaar) [reply lib/moon/design/tag.ex]: Valid point. Functions now replaced.

**Review by phgrey — APPROVED** [review]

**Pull request merged**